source/slang/slang-options.cpp

```diff
@@ -1925,6 +1957,7 @@ SlangResult OptionsParser::_parse(
                 linkage->m_optionSet.set(optionKind, compressionType);
                 break;
             }
+            case OptionKind::EmbedDXIL: SLANG_RETURN_ON_FAIL(addEmbeddedLibrary(CodeGenTarget::DXIL, CompilerOptionName::EmbedDXIL, "dxil_lib")); break;
```

Comment:
We should pass in a capability directly here.
```suggestion
            case OptionKind::EmbedDXIL: SLANG_RETURN_ON_FAIL(addEmbeddedLibrary(CodeGenTarget::DXIL, CompilerOptionName::EmbedDXIL, CapabilityName::dxil_lib)); break;
```

Reply:
Not needed in latest change